opentreemap/treemap/models.py

```diff
@@ -593,6 +590,15 @@ def __init__(self, *args, **kwargs):
         self._do_not_track.add('feature_type')
         self._do_not_track.add('mapfeature_ptr')
         self._do_not_track.add('hide_at_zoom')
+
+        # Tells the permission system that if any other field in the role
+        # is writable, _joint_writable fields are also writable.
```

Comment:
If you agree with my proposals above, we can remove the first two comment lines.